### PR TITLE
Add docker stats to compose

### PR DESCRIPTION
**Pull request by jkp85**
Signed-off-by: Jan Kowalski <jankowalskipraca85@gmail.com>

### CODE REVIEW

File: actions/middleware.py [hunk 1]
```diff
@@ -35,22 +35,17 @@ def __call__(self, request: HttpRequest) -> HttpResponse:
...
+            action=self._get_action_name(request),
+            method=request.method.lower(),
+            user_agent=request.META.get('HTTP_USER_AGENT', ''),
+            start_date=start,
+            payload=body,
+            ip=self._get_client_ip(request),
+            state=Action.PENDING,
+        )
+        action = Action.objects.get_or_create_action(filter_kwargs, defaults)
+        request.action = action
```
Comment (jgwerner) [inline actions/middleware.py]: Code Climate alerted me for a trailing white space here.

File: actions/serializers.py [hunk 2]
```diff
@@ -11,28 +11,41 @@ class ActionSerializer(serializers.ModelSerializer):
...
             'object_id': {'write_only': True, 'required': False},
         }
 
     def create(self, validated_data):
+        action_name = validated_data.pop('action_name', '')
         state = validated_data.pop('get_state_display', 0)
         content_type = validated_data.pop('content_type', None)
         user = self.context['request'].user
-        instance = Action(user=user, state=int(state), **validated_data)
+        path = validated_data.pop('path', '')
```
Comment (jgwerner) [inline actions/serializers.py]: Another trailing white space

Comment (jgwerner) [inline actions/serializers.py]: Trailing white space

File: actions/tests/test_views.py [hunk 3]
```diff
@@ -55,6 +55,7 @@ def test_create_action(self):
         url = reverse('action-create')
         action_content_object = ProjectFactory()
         data = dict(
+            action_name='detail',
```
Comment (jgwerner) [inline actions/tests/test_views.py]: Just curious, but is there a reason detail has single quotes and the other ones have double quotes?

Comment (jkp85) [inline actions/tests/test_views.py]: My rule of thumb is that user facing strings (ex. help messages) are in double quotes and other strings are single quotes. This isn't a general rule though.

Comment (jgwerner) [inline actions/tests/test_views.py]: Got it, thanks for clarifying!

**Pull request merged**